draft-ietf-rtcweb-jsep.xml

```diff
@@ -1197,7 +1197,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             gathering phase. If the policy is being made more liberal (i.e.,
             more permitted candidates), the 'needs-ice-restart' bit will be set.
             If the policy is being made more strict, no ICE restart is needed,
-            and the bit will not be set. Either way, changes to the policy have
+            and the bit will not be set. Any candidates which have already been
```

Comment:
Do you want to add a clause explaining the rationale here?

e.g. "as any relevant information has already been revealed"